static/workspace.html

```diff
@@ -81,8 +81,8 @@
               <a class="dropdown-item" id="example-onepluslambda" href="#">
                 1+λ (TODO)
               </a>
-              <a class="dropdown-item" id="example-onelambda" href="#">
-                1,λ (TODO)
+              <a class="dropdown-item" id="example-onelambda" >
+                1,λ 
```

Comment:
After a quick research, we shouldn't remove `href="#"`:

https://stackoverflow.com/questions/4855168/what-is-href-and-why-is-it-used

Reply:
In this case we should put it back in. Maybe in its own PR, because the href is missing in every implemented example.